http3/request_stream.go

```diff
@@ -0,0 +1,402 @@
+package http3
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go"
+	"github.com/lucas-clemente/quic-go/quicvarint"
+	"github.com/marten-seemann/qpack"
+)
+
+// isClientBidi returns true if id is a client-initiated bidirectional stream ID.
+func isClientBidi(id uint64) bool {
+	return id&0b11 == 0
+}
+
+type requestStream struct {
```

Comment:
Where does this come from? I assume some renamed file?

Reply:
The `requestStream` type is entirely new.